web/src/main/java/org/openmrs/web/filter/GZIPResponseWrapper.java

```diff
@@ -111,8 +111,6 @@ public void sendError(int error, String message) throws IOException {
 		super.sendError(error, message);
 		this.error = error;
 		
-		if (log.isDebugEnabled()) {
-			log.debug("sending error: " + error + " [" + message + "]");
-		}
+		log.debug("sending error: {} []", error, message);
```

Comment:
```suggestion
		log.debug("sending error: {} [{}]", error, message);
```
You still have a problem here.